src/functionalTest/java/uk/gov/hmcts/darts/courthouses/CourthousesFunctionalTest.java

```diff
@@ -95,6 +97,7 @@ void updateCourthouseWithInvalidBody() {
     }
 
     @Test
+    @Disabled
```

Comment:
It would be better if we could extract the courthouse id from the `POST /courthouses` and then use it for the `DELETE /courthouses/{courthouse_id}` operation here